### PR TITLE
Make sure the user knows our calculations are correct

### PR DESCRIPTION
**Pull request by eteq**
Have you ever watched [The Martian](https://en.wikipedia.org/wiki/The_Martian_(film)) and wished that you could have that fancy high-tech "CALCULATIONS CORRECT" screen pop up?  With this PR, now we can!  You can see a demo of it below.

output

I'm sure this will have a positive effect on the community's confidence in Astropy.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [x] Do the proposed changes actually accomplish desired goals?
- [ ] Do the proposed changes follow the [Astropy coding guidelines](https://docs.astropy.org/en/latest/development/codeguide.html)?
- [ ] Are tests added/updated as required? If so, do they follow the [Astropy testing guidelines](https://docs.astropy.org/en/latest/development/testguide.html)?
- [ ] Are docs added/updated as required? If so, do they follow the [Astropy documentation guidelines](https://docs.astropy.org/en/latest/development/docguide.html#astropy-documentation-rules-and-guidelines)?
- [ ] Is rebase and/or squash necessary? If so, please provide the author with appropriate instructions. Also see ["When to rebase and squash commits"](https://docs.astropy.org/en/latest/development/when_to_rebase.html).
- [ ] Did the CI pass? If no, are the failures related? If you need to run daily and weekly cron jobs as part of the PR, please apply the `Extra CI` label.
- [ ] Is a change log needed? If yes, did the change log check pass? If no, add the `no-changelog-entry-needed` label. If this is a manual backport, use the `skip-changelog-checks` label unless special changelog handling is necessary.
- [ ] Is this a big PR that makes a "What's new?" entry worthwhile and if so, is (1) a "what's new" entry included in this PR and (2) the "whatsnew-needed" label applied?
- [x] Is a milestone set? Milestone must be set but `astropy-bot` check might be missing; do not let the green checkmark fool you.
- [ ] At the time of adding the milestone, if the milestone set requires a backport to release branch(es), apply the appropriate `backport-X.Y.x` label(s) *before* merge.

### CODE REVIEW

File: astropy/utils/modal_utils.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+__all__ = 'make_modal'
```
Comment (pllim) [inline astropy/utils/modal_utils.py:4]: Isn't `__all__` supposed to be a list or is this some tkinter magic?
```suggestion
__all__ = ['make_modal']
```

**Comment by kelle** [comment]
This looks like a great enhancement and I know I always appreciate reassurances like this. One suggestion: I think it would be even better if the font color of the popup was a bright green.

File: astropy/utils/modal_utils.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+try:
+    import tkinter
+    from tkinter import ttk
+except ImportError:
+    tkinter = None
+
+if tkinter is None:
+    def make_modal(text, title, blink_ms, geometry, font, center_window=True):
+        print(text)
```
Comment (pllim) [inline astropy/utils/modal_utils.py:14]: Maybe use this?

https://github.com/astropy/astropy/blob/64fe8fa3b15089caac67c1806f3ce648bdb73791/astropy/utils/console.py#L297

File: astropy/utils/modal_utils.py (hunk 1 above)
Comment (eteq) [reply astropy/utils/modal_utils.py:4]: oh good catch!

**Comment by pllim** [comment]
@kelle , bright green would go against the movie lore.

Screenshot 2022-04-01 115408

**Comment by eteq** [comment]
> Should we make it even bigger if we detect that user is directly connected to JPL data center?

Yes, but definitely only if it's a direct physical connection.  No SSH allowed.

**Comment by eteq** [comment]
Oh, and I meant to highlight for @adrn's benefit that I implement this in a highly-descriptive module name: `utils/modal_utils.py`

File: astropy/utils/modal_utils.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+else:
+    def make_modal(text, title, blink_ms, geometry, font, center_window=True):
```
Comment (dhomeier) [inline astropy/utils/modal_utils.py:16]:
```suggestion
    def make_modal(text='CALCULATIONS\INCORRECT',
                   title='Are you sitting in front of a computer?',
                   blink_ms=750, geometry='550x155', font=('Helvetica', 48), color='r',
                   center_window=True):
```
https://github.com/astropy/nothing-to-see-here/issues/7#issuecomment-432301710 made a very pertinent point about sensible defaults.

**Comment by pllim** [comment]
Actually, talking about colors, I think the text should be grayish, not pure white.

**Comment by pllim** [comment]
Also, does this work in a notebook? 😉

**Comment by pllim** [comment]
My brain's sayin' let's go
But my heart is sayin' no

**Pull request closed without merging**